src/documents/management/commands/document_consumer.py

```diff
@@ -32,13 +29,22 @@ def __init__(self, *args, **kwargs):
 
         BaseCommand.__init__(self, *args, **kwargs)
 
+    def add_arguments(self, parser):
+        parser.add_argument("directory", default=settings.CONSUMPTION_DIR, nargs='?')
+        parser.add_argument("--loop-time", default=settings.CONSUMER_LOOP_TIME, type=int)
+        parser.add_argument("--mail-delta", default=10, type=int)
+        parser.add_argument("--oneshot", action='store_true')
+
     def handle(self, *args, **options):
 
         self.verbosity = options["verbosity"]
+        directory = options['directory']
```

Comment:
Please stick to using `"` for strings unless those strings contain a `"` themselves